recipes/sagemath-db-elliptic-curves/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+export SAGE_SHARE="$PREFIX/share"
+ln -s . src
```

Comment:
I'm using the python script below. `spkg-install`
There the script expects the contents of the archive to be put into `src`. It also expects `SAGE_SHARE` to be set. I've simulated both above.
I could also bundle it in the recipe dir and make a few changes, but I thought it would be easier for the future.